pages/productAppExtension/[productId]/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { H4, Panel, Text } from "@bigcommerce/big-design";
+import { useRouter } from "next/router";
+import ErrorMessage from "@components/error";
+import Loading from "@components/loading";
+import { useProductInfo, useProductList } from "@lib/hooks";
+
+const ProductAppExtension = () => {
+    const router = useRouter();
+    const productId = Number(router.query?.productId);
+    const { error, isLoading, list = [] } = useProductList();
+    const { isLoading: isInfoLoading, product } = useProductInfo(productId, list);
+    const { description, is_visible: isVisible,name, price, type,} = product ?? {};   
+    const productType = type?.replace(/^\w/, (c: string) => c.toUpperCase());
```

Comment:
🍹 I'd recommend renaming `productType` to be more descriptive of what this variable actually holds, for readability purposes. Consider naming it `typeCapitalized`, for example.